bindata/etcd/pod.yaml

```
@@ -311,6 +311,30 @@ ${COMPUTED_ENV_VARS}
         name: log-dir
       - mountPath: /etc/kubernetes/static-pod-certs
         name: cert-dir
+  - name: etcd-backup-server
+    image: ${OPERATOR_IMAGE}
+    imagePullPolicy: IfNotPresent
+    terminationMessagePolicy: FallbackToLogsOnError
+    command: [cluster-etcd-operator]
+    args:
+${COMPUTED_BACKUP_VARS}
```

this fails, because there's nothing that will populate it:
> {"app":"installer","container":"installer","host":"ip-10-0-112-138.us-east-2.compute.internal","pod":"installer-1-retry-11-ip-10-0-112-138.us-east-2.compute.internal","_entry":"F0729 14:05:26.715166       1 cmd.go:105] failed to copy: yaml: line 586: could not find expected ':'"}

```suggestion
    args: []
```

---

ach jaaa .. we dont have `backupvar` here yet 👍🏽

---

fixed ...